frontend/packages/data-portal/app/components/Dataset/DatasetMetadataTable.tsx

```diff
@@ -5,44 +5,28 @@ import { AccordionMetadataTable } from 'app/components/AccordionMetadataTable'
 import { AuthorLegend } from 'app/components/AuthorLegend'
 import { AuthorInfo } from 'app/components/AuthorLink'
 import { AuthorList } from 'app/components/AuthorList'
-import { DatabaseEntry } from 'app/components/DatabaseEntry'
+import { DatabaseEntryList } from 'app/components/DatabaseEntry'
 import { Link } from 'app/components/Link'
 import { useI18n } from 'app/hooks/useI18n'
 import { getTableData } from 'app/utils/table'
 
 import { DatasetType } from './type'
 
-function DatabaseEntryList({ entries }: { entries: string }) {
-  return (
-    <ul className="flex flex-col gap-sds-xs text-sds-body-s leading-sds-body-xs">
-      {entries.split(',').map((entry) => {
-        return (
-          <li key={entry}>
-            <DatabaseEntry entry={entry.trim()} inline />
-          </li>
-        )
-      })}
-    </ul>
-  )
-}
-
-interface DatasetMetadataTableProps {
-  dataset: DatasetType
-  allFields?: boolean
-  initialOpen?: boolean
-}
-
 export function DatasetMetadataTable({
   dataset,
   allFields,
   initialOpen,
-}: DatasetMetadataTableProps) {
+}: {
+  dataset: DatasetType
+  allFields?: boolean
```

Comment:
nit - can this have a name that indicates it's a boolean?  Maybe `showAllFields`?